{
  "affected_release": null,
  "package_state": [
    {
      "product_name": "Red Hat Enterprise Linux 8",
      "fix_state": "Affected",
      "package_name": "python-cryptography",
      "cpe": "cpe:/o:redhat:enterprise_linux:8"
    }
  ],
  "threat_severity": "Moderate",
  "public_date": "2020-10-25T00:00:00Z",
  "bugzilla": {
    "description": "CVE-2020-25659 python-cryptography: bleichenbacher timing oracle attack against RSA decryption",
    "id": "1889988",
    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=1889988"
  },
  "cvss": {
    "cvss_base_score": "",
    "cvss_scoring_vector": "",
    "status": ""
  },
  "cvss3": {
    "cvss3_base_score": "7.5",
    "cvss3_scoring_vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:N/A:N",
    "status": "draft"
  },
  "iava": "",
  "cwe": "CWE-385",
  "statement": "",
  "acknowledgement": "",
  "name": "CVE-2020-25659",
  "document_distribution": "",
  "details": [
    "No description is available for this CVE."
  ],
  "references": null
}